tools/downloader/topologies_test.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+import os
+import numpy as np
+import cv2 as cv
+
+import topologies as omz
+
+from pathlib import Path
+from parameterized import parameterized
+from openvino import inference_engine
+from openvino.inference_engine import IECore
```

Comment:
Please reorder your imports to the order suggested by [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports).